lib/database.js

```diff
@@ -1149,7 +1149,7 @@ function Database(cn, dc, config) {
          * @example
          *
          * db.each('SELECT id, code, name FROM Events', [], row => {
-         *     row.code = +row.code; // leading `+` is short for `parseInt()`
+         *     row.code = row.code|0; // trailing `|0` is short for `parseInt()`
```

Comment:
I understand that `| 0` is indeed the shorter version, but in this case I'd rather use `parseInt` in documentation, not to confuse developers with some strange-looking syntax :smile:

Reply:
it's not strange-lloking syntaxt :rofl: